system/ee/legacy/libraries/Cache/drivers/Cache_redis.php

```diff
@@ -179,7 +179,8 @@ protected function _setup_redis()
             'host' => '127.0.0.1',
             'password' => null,
             'port' => 6379,
-            'timeout' => 0
+            'timeout' => 0,
+            'database' => 0
```

Comment:
Is there a reason why the default is integer and not empty string or null?

Reply:
Good question, the default setup for Redis involves 16 databases that have IDs 0-15.  The default database value will be set to 0 to use the first one available.